internal/flink/command_artifact_create.go

```diff
@@ -58,10 +65,15 @@ func (c *command) createArtifact(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	runtimeLanguage, err := cmd.Flags().GetString("runtime-language")
+	if err != nil {
+		return err
+	}
 
 	extension := strings.TrimPrefix(filepath.Ext(artifactFile), ".")
-	if strings.ToLower(extension) != "jar" {
-		return fmt.Errorf(`only ".jar" file extensions are allowed`)
+	ok := slices.Contains(allowedFileExtensions, extension)
```

Comment:
The previous implementation uses `strings.ToLower()` so this looks like it might be a regression.